.github/workflows/ci-suite.yml

```diff
@@ -1,6 +1,8 @@
 name: 'CI'
 
 on:
+  schedule:
+    - cron: 0 0 * * *
```

Comment:
Schedule this for 23:00

CRON runs at 00:00 are so backlogged ours end up completing at 01:30.